functions/helloworld/helloworldHttp/test/sample.unit.http.test.js

```diff
@@ -13,7 +13,10 @@
 // limitations under the License.
 
 // FF testing layer for declarative signatures
+
+// [START functions_http_unit_test]
```

Comment:
dupe of region tag below. did you mean to name this something differently?

Reply:
Region tags merge together if they have the same naming.